### PR TITLE
Fix Default ES TCP Port

### PR DESCRIPTION
**Pull request by svairagade**
Updated ES TCP port to 1113

https://eventstore.com/docs/server/ports-and-networking/index.html

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ The `Unreleased` section name is replaced by the expected version of next releas
 ### Changed
 
 - DRY'd handling of `MissingArg` in `Configuration.initialize`
+- `proReactor`: Fixed Default ES TCP port to 1113 [#69](https://github.com/jet/dotnet-templates/pull/69)
```
Comment (bartelink) [inline CHANGELOG.md:42]: needs to move to Unreleased section up top

File: propulsion-reactor/Program.fs [hunk 2]
```diff
@@ -357,7 +357,7 @@ module Args =
             match tcp, port with
             | false, None ->   Discovery.GossipDns            host
             | false, Some p -> Discovery.GossipDnsCustomPort (host, p)
-            | true, None ->    Discovery.Uri                 (UriBuilder("tcp", host).Uri)
+            | true, None ->    Discovery.Uri                 (UriBuilder("tcp", host, 1113).Uri)
```
Comment (bartelink) [inline propulsion-reactor/Program.fs:360]: there's a clone of this in `propulsion-sync` too

File: CHANGELOG.md (hunk 1 above)
Comment (svairagade) [reply CHANGELOG.md:42]: 🤦, will fix it.

File: propulsion-reactor/Program.fs (hunk 2 above)
Comment (svairagade) [reply propulsion-reactor/Program.fs:360]: Done

**Pull request merged**